cuav/image/include/imageutil.h

```diff
@@ -9,11 +9,25 @@
 #include <stdlib.h>
 #include <stdarg.h>
 
-#define PACKED __attribute__((__packed__))
+#ifdef _MSC_VER
```

Comment:
i think we could make macros with the same syntax for both?

Reply:
Couldn't figure out a macro that makes both gcc and Visual Studio happy